drydock/templates/kustomized/tutor13/extensions/debug/ingress.yml

```diff
@@ -0,0 +1,35 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: ingress-canary
+  namespace: {{ K8S_NAMESPACE }}
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/canary: "true"
+    nginx.ingress.kubernetes.io/canary-by-cookie: {{ OPENEDX_DEBUG_COOKIE }}
+    {%- if DRYDOCK_AUTO_TLS and not DRYDOCK_CUSTOM_CERTS%}
+    cert-manager.io/issuer: letsencrypt
+    {%- endif %}
+    argocd.argoproj.io/sync-wave: "5"
+spec:
```

Comment:
Maybe we can add a default backend that points to caddy. IIRC the controller throws an error log if it doesn't find any service to fallback.

Reply:
I'll try that. Thanks!